pyflakes/checker.py

```diff
@@ -717,14 +717,24 @@ def _bare_name_is_attr(name):
 
         return False
 
+    def _module_scope_is_typing(name):
```

Comment:
Would this be good to expose as a function that plugins can use. Then tools like `flake8-typing-imports` could use the same solution.

Reply:
no, pyflakes does not expose its stack handling nor its non-standard ast parser and it's very specialized for pyflakes